zkfold-cardano-benchmark/common/ZkFoldBenchmark/Common.hs

```diff
@@ -0,0 +1,89 @@
+module ZkFoldBenchmark.Common (TestSize (..), printHeader, printSizeStatistics) where
+
+import qualified Data.ByteString                                   as BS
+import           Data.SatInt                                       (fromSatInt)
+import qualified Flat
+import           PlutusCore.Default                                (DefaultFun, DefaultUni)
+import           PlutusCore.Evaluation.Machine.ExBudget            (ExBudget (..))
+import qualified PlutusCore.Evaluation.Machine.ExBudgetingDefaults as PLC
+import           PlutusCore.Evaluation.Machine.ExMemory            (ExCPU (..), ExMemory (..))
+import           Prelude
+import           System.IO                                         (Handle)
+import           Text.Printf                                       (hPrintf, printf)
+import qualified UntypedPlutusCore                                 as UPLC
+import qualified UntypedPlutusCore.Evaluation.Machine.Cek          as Cek
+
+-- Protocol parameters (June 2023)
+
+-- | This is the "maximum transaction size".  We're just comparing the size of
+-- the script with this, so our results may be a little optimistic if the
+-- transaction includes other stuff (I'm not sure exactly what "maximum
+-- transaction size" means).
```

Comment:
We do not need to include the script in a transaction. Since PlutusV2, we can include a reference UTxO with the script in the datum. This way, it will not count towards the transaction size.

However, it still should be less than the maximal transaction size so that we can create such a UTxO in the first place.